recipe/index.md

```diff
@@ -12,32 +12,54 @@ The [IIIF Presentation API][prezi3] specifies a standardised way to describe com
 * enable client software developers to support these patterns, for consistency of user experience (when desirable);
 * demonstrate the applicability of IIIF to a broad range of use cases.
 
+The cookbook will eventually be part of the main iiif.io site, but is currently a separate previewable site:
+
+[https://preview.iiif.io/cookbook/master/recipe/](https://preview.iiif.io/cookbook/master/recipe/)
+
+The cookbook-recipes repository is a Jekyll website, like the main IIIF site, with [some additions](https://github.com/IIIF/cookbook-recipes/#jekyll-variables) to make it easier to manage recipes and their dependent parts. In the IIIF specifications site, JSON-LD examples are embedded snippets in the page markdown. But in the recipes repository, code must be complete and standalone - we need our recipes to be complete working examples, not just extracts for guidance. Typically a recipe will include an entire manifest, as a separate JSON-LD file, that points to real assets, that works in a viewer. 
+
 # Process
 
 Anyone can submit a recipe to the cookbook. The community process is as follows:
 
 * The proposer creates an issue for the recipe on GitHub.
-* The initial discussion of the recipe is conducted in comments on the issue. This may result in the recipe being modified, or abandoned.
+* The initial discussion of the recipe is conducted in comments on the issue and through the Cookbook channel on the [IIIF Slack](). This may result in the recipe being modified, or abandoned.
 * If there is broad agreement that the recipe is valuable, the proposer, or another community member assigns the issue to themselves. 
   * To get permission to assign issues please contact the IIIF Staff at admin@iiif.io who will add you to the iiif-recipes Github team. 
-* The assignee will then create a pull request containing the recipe.
-* The pull request is reviewed by the Technical Review Committee as part of its regular work.
-* If approved, the pull request is merged, and the recipe becomes part of the published cookbook.
+* Clone the cookbook repository
+* Create a new branch locally to work on your recipe
+* The assignee will then create a draft pull request
+  * If you host the branch on the IIIF/cookbook repository the pull request build process will build a preview site. 
+* Once the assignee has completed their work on the pull request they will move it to a public pull request 
+* Review process
+  * Informal review by Cookbook group (announce on Cookbook slack channel)
+  * First review by Cookbook Editors group
+  * Final approval by the Technical Review Committee and after this the pull request will be merged to master)
```

Comment:
Does it make sense to explain what the process is like if the proposed recipe does not pass reviews at any of these stages?  Does it get informally reviewed again by the Cookbook group as the first step after receiving any feedback for changes and then proceed forward again?